vue.config.js

```diff
@@ -23,14 +23,22 @@ module.exports = {
   publicPath: '',
   outputDir: 'dist',
   indexPath: 'index.html',
-  transpileDependencies: ['vuetify'],
+  transpileDependencies: ['vuetify', 'graphql-language-service-interface', 'graphql-language-service-parser'],
   runtimeCompiler: true,
   productionSourceMap: process.env.NODE_ENV !== 'production',
   pluginOptions: {
     apollo: {
       lintGQL: false
     }
   },
+  devServer: {
+    proxy: {
+      '^/userprofile': {
+        target: 'http://localhost:3000/',
```

Comment:
Is 3000 the default of the json-server?

Reply:
Yes.

![image](https://user-images.githubusercontent.com/304786/118410041-a0680300-b6e1-11eb-8b82-82f4b05845c9.png)

But doesn't hurt to be explicit about the port in our `scripts` section (also helpful if `3000` is in use and we want to quickly switch to a different port locally). Pushed a new commit with the port in the `scripts` section.

![image](https://user-images.githubusercontent.com/304786/118410089-d311fb80-b6e1-11eb-87a8-9538d07722cb.png)